### ISSUE TITLE
Synced tabs aren't displayed after sign out/sign in

### ISSUE THREAD

**Issue by lougeniaC64** [mozilla-mobile/firefox-ios#12357]
This is an iOS tracking bug for [Bugzilla bug no. 1799077](https://bugzilla.mozilla.org/show_bug.cgi?id=1799077).

┆Issue is synchronized with this [Jira Task](https://mozilla-hub.atlassian.net/browse/FXIOS-5233)

**Comment by lmarceau** [comment]
This is a duplicate of https://github.com/mozilla-mobile/firefox-ios/issues/12315 so I'll close it in favor of the other one. Thanks!

**Comment by lmarceau** [comment]
Turns out it's not a duplicate, it was 1799097 instead of 1799077 😬

**Comment by data-sync-user** [comment]
➤ Nishant Bhasin commented:

https://github.com/mozilla-mobile/firefox-ios/pull/12358#pullrequestreview-1179923349 ( https://github.com/mozilla-mobile/firefox-ios/pull/12358#pullrequestreview-1179923349 )

**Comment by data-sync-user** [comment]
➤ Nishant Bhasin commented:

Waiting on builds to go green before I merge

**Comment by data-sync-user** [comment]
➤ Simion Basca commented:

Verifying as fix on v108 (22758)
